docs/tips-and-tricks.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+  <h2 align="center">📖 Tips and tricks for writing a userstyle</h2>
+</p>
+
+&nbsp;
+
+### How can I see my changes in real time?
+
+If you want to see your changes in real-time while using an external editor:
+
+1. Head to `chrome://extensions/` or your browser's equivalent.
```

Comment:
We could update https://github.com/catppuccin/userstyles/blob/main/docs/browsers.md and link to?

Reply:
This makes sense.